flask/namespaces.py

```diff
@@ -4,6 +4,46 @@
 from app import swagger
 from flasgger.utils import swag_from
 
+from datetime import datetime
+from auth import generate_uuid
+
+@app.route("/namespaces", methods=["POST"])
+def create_namespace():
+    uuid = request.form.get("uuid")
+
+    if not uuid:
+        return jsonify({"code": 401, "message": "Unauthorized"}), 401
+    
+    namespace_name = request.form.get("namespace")
+
+    if not namespace_name:
+        return jsonify({"code": 400, "message": "Please enter namespace name"}), 400
+    
+    # Get the user document from the uuid.
+    user_doc = db.users.find_one({"uuid": uuid})
+
+    # Get the namespace document from the namespace name.
+    # To check if already a namespace exists by this name.
+    namespace_doc = db.namespaces.find_one({"namespace": namespace_name})
```

Comment:
This seems to be the only check we're doing. Can I now upload an empty string a namespace? Or a blank? Or "g3p9i-.1")$(". Maybe we should do some format checking.

Reply:
I think every other string except blank should work. But for implementing the checks for naming of namespace i think I need some more info from you & @perazz how fpm users should be allowed to name the namespaces and then I think it should be easier to implement. But, I think for now we can trust the users that they will give the namespace a relevant name 😓

Reply:
I'm not sure but I'd only allow `a-z`, `A-Z`, `0-9`, `-` and `_`. Not sure if we should take both of the last two.

Reply:
I believe package namespaces don't need to be used by Fortran code, so I'd say there is more freedom than allowed for the module names in the code. Like @minhqdao suggests, avoiding special characters and spaces should just work